pkg/credentialwatcher/secretwatcher.go

```diff
@@ -139,20 +139,24 @@ func (s *secretWatcher) updateAccountRotateCredentialsStatus(log logr.Logger, ac
 		return
 	}
 
-	if accountInstance.Status.RotateCredentials != true {
-
-		if credentialType == "console" {
-			accountInstance.Status.RotateConsoleCredentials = true
-			log.Info(fmt.Sprintf("AWS console credentials secret was created ago requeueing to be refreshed"))
-		} else if credentialType == "cli" {
-			accountInstance.Status.RotateCredentials = true
-			log.Info(fmt.Sprintf("AWS cli credentials secret was created ago requeueing to be refreshed"))
-		}
+	if (accountInstance.Status.RotateConsoleCredentials && credentialType == "console") || (accountInstance.Status.RotateCredentials && credentialType == "cli") {
```

Comment:
Can we simplify this into functions similar to what @clcollins did?  It took me a bit of time to parse what we were checking for here.

Reply:
We'll open a new card for this refactor so we can add tests and track appropriately.